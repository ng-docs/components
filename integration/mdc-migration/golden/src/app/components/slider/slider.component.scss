.mat-mdc-slider {
  width: 50%;
}

/* TODO(mdc-migration): The following rule targets internal classes of slider that may no longer apply for the MDC version. */
.mat-slider-thumb { background-color: transparent; }
