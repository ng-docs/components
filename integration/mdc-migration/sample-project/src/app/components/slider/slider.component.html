<h2>Slider example</h2>
<mat-slider></mat-slider>
