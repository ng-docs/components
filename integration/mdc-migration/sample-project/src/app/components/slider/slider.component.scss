.mat-slider {
  width: 50%;
}

.mat-slider-thumb { background-color: transparent; }
