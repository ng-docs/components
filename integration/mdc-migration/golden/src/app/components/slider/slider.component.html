<h2>Slider example</h2>
<mat-slider><input matSliderThumb /></mat-slider>
